gatsby-browser.tsx

```diff
@@ -1,12 +1,16 @@
 import React from 'react';
 import {GatsbyBrowser} from 'gatsby';
 
+import {FeedbackWidget} from 'sentry-docs/components/feedback/feedbackWidget';
 import PageContext from 'sentry-docs/components/pageContext';
 
 export const wrapPageElement: GatsbyBrowser['wrapPageElement'] = ({
   element,
   props: {pageContext},
-}) => <PageContext.Provider value={pageContext}>{element}</PageContext.Provider>;
+}) => <PageContext.Provider value={pageContext}>
+        <FeedbackWidget />
+        {element}
+    </PageContext.Provider>
```

Comment:
Don't we need to do this in `gatsby-node` as well?

Reply:
Ahh yeah probably, nice catch